src/test/resources/end2end_test_docker_curl_commands.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
```

Comment:
I might be wrong but it seems to me that this file is not used in the end2end tests. We can remove this file, What do you think?

Reply:
The same as with the kie compose. It was helpful to have it in order to recreate the infrastracture with the compose. We can remove it , but I thought it was interesting to have it.